windows/src/developer/TIKE/xml/kmw/index.html

```diff
@@ -156,7 +153,15 @@
   <body class='osk-always-visible'>
     <div class='header'>
       <div><h2>Keyboard Test</h2></div>
```

Comment:
Do you want a general title now that the page is for testing Keyboard and Models?

Reply:
I think I'll just keep the existing title -- model testing doesn't happen in isolation to the keyboard testing.